lib/team.rb

```diff
@@ -8,96 +8,89 @@ class Team
               :abbreviation,
               :link
 
-attr_accessor :home_wins,
-              :away_wins,
-              :total_wins,
-
-              :away_goals,
-              :home_goals,
-              :total_goals,
-
-              :away_games,
-              :home_games,
-              :total_games,
-
-              :total_goals_allowed,
-
-              :preseason_wins,
-              :preseason_games,
-              :preseason_goals,
-              :preseason_goals_allowed
+  attr_accessor :home,
```

Comment:
I think this way is ok. Feels harder to read/understand to me. It does total the regular and preseason games which wasn't happening before which is good. I'm worried that starting to put more and more info into hashes (which we want to do for seasons) is going to bring us closer to where we started initially, which was digging into layers of hashes which is complicated to write/access/read.